cmd/pd-server/main.go

```diff
@@ -281,6 +282,10 @@ func start(cmd *cobra.Command, args []string, services ...string) {
 		serviceBuilders = append(serviceBuilders, swaggerserver.NewHandler)
 	}
 	serviceBuilders = append(serviceBuilders, dashboard.GetServiceBuilders()...)
+	if len(services) > 0 && !cfg.Microservice.IsResourceManagerFallbackEnabled() {
```

Comment:
Why enable the redirector when `IsResourceManagerFallbackEnabled` is false?

Reply:
This parameter name is a bit confusing. IsResourceManagerFallbackEnabled actually refers to DisableResourceManager. However, since serverless already uses this parameter, we'll refrain from modifying it.

Fallback means fallback from ResourceManager to PD